src/orion/core/__init__.py

```diff
@@ -73,6 +74,17 @@ def define_config():
     return config
 
 
+def define_frontends_config(config):
+    """Create and define the field of frontends configuration."""
+    config.add_option(
+        "frontends",
```

Comment:
I call the option something like frontends_uri or something like this so that we know more precisely what it is.

I think an env_var like `ORION_WEBAPI_FRONTENDS_URI` would be useful.